src/xform_to_datalog/generate_datalog_program.cc

```diff
@@ -71,10 +72,11 @@ int main(int argc, char* argv[]) {
 
   auto datalog_facts = raksha::xform_to_datalog::DatalogFacts(
       raksha::xform_to_datalog::ManifestDatalogFacts::CreateFromManifestProto(
-          manifest_proto));
+          manifest_proto),
+      raksha::xform_to_datalog::AuthorizationLogicDatalogFacts(""));
```

Comment:
This will be filled in with a non-empty string in a follow-on PR?

Reply:
Yes, I have a PR that will fill this in. Without this change, the tests won't pass.